templates/client.ts.hbs

```diff
@@ -12,6 +12,9 @@ import ResponseError from "./responseError";
 import TemplateURL from "./templateUrl";
 import { USER_AGENT_HEADER, USER_AGENT_VALUE } from "./version";
 
+// The goal is to run native fetch in the 'real' browser. The reason isNode is also checked so to not use the fetch in jest-dom (nock tests) or other browser mocking libraries.
+const [runFetchStrategy, ValidRequest] = isBrowser && hasFetchAvailable && !isNode ? [fetch, Request] : [crossFetch, CrossRequest];
```

Comment:
Yes, I did verify manually in the browser. 

This is the current state of the fetch call. Notice the red request (right now it is failing because of wrong credentials but notice the type and initiator)
![image](https://user-images.githubusercontent.com/88844579/224361597-60584779-e661-47e6-a9e7-064db9db8413.png)

If the cross fetch called, it would say this:
<img width="1504" alt="Screenshot 2023-03-10 at 10 53 46 AM" src="https://user-images.githubusercontent.com/88844579/224362345-83bf8351-5c2e-4dc6-916e-e83fcb627dd1.png">

Reply:
Awesome! Were you able to verify it in a node environment?

Reply:
Not manually, no. I thought the tests already cover the node environment (and the react test is also in node environment because it mocks the browser)